src/tag/tag-filter.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, HostBinding } from "@angular/core";
+import { Tag } from "./tag.component";
+
+@Component({
+	selector: "ibm-tag-filter",
+	template: `
+		<ng-content></ng-content>
+		<svg
```

Comment:
No icon module?

Reply:
I even wrote this one ... the close icon is targeted with `>` so right now icons-angular won't work